### DISCUSSION TITLE
Trigger validation for list only when adding/removing

### DISCUSSION THREAD

**Discussion by deanCompRSA** [vueform/vueform#330] [Questions] [answered]
Setting up the validate like this works, but also validates the elements inside the list which I do not want to happen. How can I prevent this?
```
myList: {
  onChange: (_: any, __: any, element: any) => {element.validate}
}
```

**Comment by adamberecz** [comment, marked as answer]
Probably [validateValidators()](https://vueform.com/reference/list-element#method-validate-validators) method is what you're looking for.

**Reply by deanCompRSA** [reply]
My man
